spec/game_session_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'game_session'
+require 'pry'
+
+describe GameSession do
```

Comment:
When running the test "Welcome to TicTacToe! Player 1 is 'X' and Player 2 is '0'."  is displayed to the console. The test should run without any noise. Can we capture the output (https://stackoverflow.com/questions/14987362/how-can-i-capture-stdout-to-a-string) before it is sent to standard out. This should also make testing the UI easier.